client/Screens/LoginScreen.js

```diff
@@ -14,7 +14,9 @@ export default class LoginScreen extends Component {
 
   loginPressHandler() {
     // AJAX request to http://server_ip/api/users/login
-    fetch('http://server_ip/api/users/login', {
+    console.log('username: ', this.state.Username);
+    console.log('password: ', this.state.Password);
+    fetch('https://todaysjam.herokuapp.com/api/users/login', {
```

Comment:
ajax request for login, will send info to the backend api and allow backend to verify authentication